opx/sai_vm_event_log.h

```diff
@@ -18,9 +18,16 @@
 #ifndef __SAI_VM_EVENT_LOG_H__
 #define __SAI_VM_EVENT_LOG_H__
 
+#ifdef __cplusplus
+extern "C"
+{
+#endif
 #include "event_log.h"
 #include "event_log_types.h"
 #include "sai_event_log.h"
+#ifdef __cplusplus
+}
+#endif
```

Comment:
This is not required. 
https://github.com/open-switch/opx-sai-vm/pull/2/commits/fc4c03a5e334f2b1a14d1aff6c6eeac95cb6328f